### PR TITLE
Added HasCoverMessage and basic config for it

### PR DESCRIPTION
**Pull request by cableman**
#### Link to ticket

Internal tickets (lots of diffrent tickets).

#### Description

Send information about cover updates to hasCover service.

#### Screenshot of the result

No screenshot.

#### Checklist

- [x] My code is covered by test cases.
- [x] My code passes our test (all our tests).
- [x] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

No comments.

### CODE REVIEW

**Review by turegjorup — CHANGES_REQUESTED** [review]

File: src/Service/HasCoverService.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+     */
+    public function post(string $pid, bool $coverExists)
+    {
+        $labels = [
+            'type' => 'hasCover',
+        ];
+
+        try {
+            $response = $this->client->request('POST', $this->hasCoverServiceUrl, [
+                'auth_bearer' => $this->authenticationService->getAccessToken(),
```
[the remark this answers is not in the document]
Comment (cableman) [reply src/Service/HasCoverService.php:54]: yes :-D

File: src/Repository/SearchRepository.php [hunk 2]
```diff
@@ -60,4 +61,38 @@ public function getNumberOfRecords(): int
...
+     *   The offset to start at
+     *
+     * @return query
+     *   The query build
+     */
+    public function findSearchesByType(string $type, ?string $identifier, int $limit, int $offset): Query
+    {
+        $queryBuilder = $this->createQueryBuilder('s');
+        $queryBuilder->select('s')
+            ->where('s.isType = \''.$type.'\'');
```
[the remark this answers is not in the document]
Comment (cableman) [reply src/Repository/SearchRepository.php]: Fixed

File: .env [hunk 3]
```diff
@@ -64,6 +64,8 @@ APP_NO_HITS_TTL=3600000
 CLOUDINARY_CLOUD_NAME=myCloudName
 CLOUDINARY_API_KEY=MyCloudApiKey
 CLOUDINARY_API_SECRET=MyCloudApiSecret
+
+HAS_COVER_SERVICE_URL=https://ddfhascover-stg.dbc.dk/api/v1/events
```
[the remark this answers is not in the document]
Comment (cableman) [reply .env:68]: In some places they call it DDF has cover as it's a service only used by DDF.

**Review by turegjorup — APPROVED** [review]

**Comment by cableman** [comment]
Rebase once more with development to match newest code base.

File: infrastructure/cover-service-cron-jobs/values.yaml [hunk 4]
```diff
@@ -20,6 +20,8 @@ app:
     imagePullPolicy: IfNotPresent
   indexing:
     url: "http://es-elasticsearch-coordinating-only.cover-service.svc.cluster.local:9200"
+  has_covere:
```
Comment (turegjorup) [inline infrastructure/cover-service-cron-jobs/values.yaml]: Shoudn't this be `has_cover`?

File: infrastructure/cover-service-importers/templates/configmap.yaml [hunk 5]
```diff
@@ -22,6 +22,7 @@ data:
   APP_METRICS_HOST: {{ .Values.app.redis.server | quote }}
   APP_METRICS_PORT: {{ .Values.app.redis.port | quote }}
   APP_INDEXING_URL: {{ .Values.app.indexing.url | quote }}
+  APP_HAS_COVER_SERVICE_URL: {{ .Values.app.has_covere.url | quote }}
```
Comment (turegjorup) [inline infrastructure/cover-service-importers/templates/configmap.yaml]: `has_cover`?

File: infrastructure/cover-service-importers/values.yaml [hunk 6]
```diff
@@ -10,6 +10,8 @@ app:
     imagePullPolicy: IfNotPresent
   indexing:
     url: "http://es-elasticsearch.cover-service.svc.cluster.local:9200"
+  has_covere:
```
Comment (turegjorup) [inline infrastructure/cover-service-importers/values.yaml]: `has_cover`?

File: infrastructure/cover-service-cron-jobs/templates/configmap.yaml [hunk 7]
```diff
@@ -21,6 +21,7 @@ data:
   APP_METRICS_HOST: {{ .Values.app.redis.server | quote }}
   APP_METRICS_PORT: {{ .Values.app.redis.port | quote }}
   APP_INDEXING_URL: {{ .Values.app.indexing.url | quote }}
+  APP_HAS_COVER_SERVICE_URL: {{ .Values.app.has_covere.url | quote }}
```
Comment (turegjorup) [inline infrastructure/cover-service-cron-jobs/templates/configmap.yaml]: Shouldn't this be `has_cover`?

**Review by turegjorup — APPROVED** [review]

**Review by turegjorup — APPROVED** [review]

**Pull request merged**